tests/test_examples.py

```diff
@@ -27,20 +31,30 @@
 ]
 
 
-def get_file_paths():
-    """Gets all the files from the examples folder for testing. This will
+def get_file_paths(paths: Union[str, List[str]]):
+    """
+    Gets all the files from the examples folder for testing. This will
```

Comment:
It's a good idea to parameterize this function, but the update to the docstring should include this first sentence. This function no longer "Gets all the files from the examples folder for testing"."